### PR TITLE
Initial global component dark theme

### PR DESCRIPTION
**Pull request by atomworks**
This PR introduces proper dark theming to global components and is formally the first step to being able to roll out a dark theme to the UI kit and for use in upcoming  projects.

Currently this is work in progress as I would like to tweak how some of the integrations are being used and polish some small details although it is essentially functional.

### CODE REVIEW

**Comment by JoshLipps** [comment]
https://github.com/future-standard/scorer-ui-kit/releases/tag/v1.8.0-beta.6
Pushed to npm on this beta version.

**Review by JoshLipps — APPROVED** [review]
This looks good for initial pass to get the base theme set.

File: src/themes/dark/styles.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { colors } from './colors';export const styles = {
```
Comment (isacoder) [inline src/themes/dark/styles.ts:1]: same line, if possible change before merge

File: src/themes/light/styles.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export const styles = {
+import { colors } from './colors';export const styles = {
```
Comment (isacoder) [inline src/themes/light/styles.ts:1]: same line, if possible change before merge

**Comment by isacoder** [comment]
Over all looks good, I understand some theming colors will stay ouside the zeppling scope and will be handle in js files however will get the values from the theme that way we can verify that all components will have css values from theme in a more obvious way.

File: src/themes/dark/styles.ts (hunk 1 above)
Comment (atomworks) [reply src/themes/dark/styles.ts:1]: For some reason the line breaks my code generates don't show up here 😭 

If you look at the raw file you can see the breaks but not here for some reason. Tried a few things but no change so I think we may have to leave it for now.

Comment (atomworks) [reply src/themes/dark/styles.ts]: Wait... no, I screwed up. The new line breaks I try work... I just didn't compile my sketch plugin after I updated it and so I was just recompiling the same thing all the time.

**Comment by JoshLipps** [comment]
This is missing many commits from the final branch used in beta.

**Comment by JoshLipps** [comment]
I think this PR should be canceled in favor of the new final branch.

**Pull request closed without merging**